CaloOnlineTools/HcalOnlineDb/interface/HcalLutGenerator.h

```diff
@@ -10,6 +10,8 @@
 #include "CalibFormats/HcalObjects/interface/HcalDbService.h"
 #include "CalibFormats/HcalObjects/interface/HcalDbRecord.h"
 #include "CondFormats/HcalObjects/interface/HcalChannelQuality.h"
+#include "CalibFormats/CaloTPG/interface/CaloTPGTranscoder.h"
+#include "CalibFormats/CaloTPG/interface/CaloTPGRecord.h"
 #include "CondFormats/DataRecord/interface/HcalChannelQualityRcd.h"
 
 class HcalLutGenerator : public edm::EDAnalyzer {
```

Comment:
As caught by the SA
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-125c12/19392/llvm-analysis/report-ea83a0.html#EndPath
this should be migrated away from being a legacy module